_docs-sources/pipelines/supported-clouds/index.md

```diff
@@ -0,0 +1,5 @@
+# Supported clouds
+
+Today, Gruntwork Pipelines works exclusively with Amazon Web Services (AWS).
+
+However, the [Gruntwork Pipelines architecture](../how-it-works) is designed to support any cloud, so we will add support for other CI systems in the future.
```

Comment:
This also isn't technically true, I'd remove this page

Reply:
Oh, that's a shame. We'll definitely want to support other clouds at some point. We do need to clarify that Pipelines only works with AWS today, but feel free to update the "designed to support any cloud" bit as needed.